test/saithrift/tests/switch.py

```diff
@@ -139,7 +139,7 @@ def sai_thrift_flush_fdb_by_vlan(client, vlan_id):
     fdb_attr_list = [fdb_attribute1, fdb_attribute2]
     client.sai_thrift_flush_fdb_entries(thrift_attr_list=fdb_attr_list)
 
-def sai_thrift_create_virtual_router(client, v4_enabled, v6_enabled):
+def sai_thrift_create_virtual_router(client, v4_enabled, v6_enabled, mac):
     #v4 enabled
```

Comment:
Understand your point. As we discussed in our meeting, we will get the tests running 1st and then I will make necessary changes to fix this issue.